test/TestCompoundMath.sol

```diff
@@ -67,27 +87,7 @@ contract TestCompoundMath is Test {
     }
 
     function testGasDiv() public view {
-        compoundMath.div(10 * WAD, WAD);
-        compoundMathRef.div(10 * WAD, WAD);
-    }
-}
-
-contract CompoundMathFunctions {
-    function mul(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMath.mul(x, y);
-    }
-
-    function div(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMath.div(x, y);
-    }
-}
-
-contract CompoundMathFunctionsRef {
-    function mul(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMathRef.mul(x, y);
-    }
-
-    function div(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMathRef.div(x, y);
+        compoundMath.div(2 * WAD, WAD);
+        compoundMathRef.div(2 * WAD, WAD);
```

Comment:
There are some lower case `compoundmath` left. We can also remove those variables and the `setUp` function

Reply:
Nope, these are used for gas report and need a contract rather than a library (because Foundry does not generate gas reports of libraries)

Reply:
Oh ok, my bad